pkg/providerconfig/types_test.go

```diff
@@ -28,32 +30,148 @@ func TestConfigVarStringUnmarshalling(t *testing.T) {
 
 func TestConfigVarBoolUnmarshalling(t *testing.T) {
 	jsonBool := []byte("true")
-	jsonString := []byte(`"true"`)
```

Comment:
I've removed the possibility to define a `configVarBool` as boolean string because IMHO there is little reason to have it and it clutters the `UnmarshalJSON` code a lot